circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  ruby:
+    version: 2.2.7
```

Comment:
I'd suggest testing with multiple Ruby versions straight away.
Given CircleCI makes that [quite painful](https://circleci.com/docs/1.0/parallel-manual-setup/#build-matrix-support), maybe just use Travis?